File: src/types/hsn-master.ts
export interface HSNMaster {
  id: string;
  name: string;
  hsnCode: string;
  gstPercentage: number;
  createdAt: Date;
  updatedAt: Date;
}

export type HSNMasterFormData = Omit<HSNMaster, "id" | "createdAt" | "updatedAt">;

File: backend/Milan.API/Models/Domain/HSNMaster.cs
using System;

namespace Milan.API.Models.Domain
{
    public class HSNMaster
    {
        public int HSNId { get; set; }
        public int TenantId { get; set; }
        public string Name { get; set; } = string.Empty;
        public string HSNCode { get; set; } = string.Empty;
        public decimal GSTPercentage { get; set; }
        public bool IsActive { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
    }
}
